### ISSUE TITLE
CPUID breaks when it returns nothing (Fedora 26)

### ISSUE THREAD

**Issue by pnemade** [workhorsy/py-cpuinfo#54]
When I tried to locally build 0.2.6 version on fedora 26, I get this

```
======================================================================
ERROR: test_all (test_actual.TestActual)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/parag/gitvcs/python-cpuinfo/master/py-cpuinfo-0.2.6/tests/test_actual.py", line 66, in test_all
    self.assertHasResult(cpuinfo.get_cpu_info_from_cpuid())
  File "/home/parag/gitvcs/python-cpuinfo/master/py-cpuinfo-0.2.6/cpuinfo/cpuinfo.py", line 935, in get_cpu_info_from_cpuid
    info = b64_to_obj(output)
  File "/home/parag/gitvcs/python-cpuinfo/master/py-cpuinfo-0.2.6/cpuinfo/cpuinfo.py", line 177, in b64_to_obj
    b = pickle.loads(a)
  File "/usr/lib64/python2.7/pickle.py", line 1388, in loads
    return Unpickler(file).load()
  File "/usr/lib64/python2.7/pickle.py", line 864, in load
    dispatch[key](self)
KeyError: '6'

----------------------------------------------------------------------
Ran 13 tests in 0.063s
```

I am using 64bit machine.

**Comment by workhorsy** [comment]
It looks like this is breaking because it is failing to CPUID, and returning None.

Thanks.

**Comment by workhorsy** [comment]
Fixed in 013c04b24bdf286657cb2a28bb691fa1e88aecb2

**Comment by pnemade** [comment]
@workhorsy Thank you.
